cdhweb/pages/models.py

```diff
@@ -157,6 +155,13 @@ def get_context(self, request):
         '''Add featured updates, projects, and events to the page context.'''
         context = super().get_context(request)
 
+        # FIXME an ImportError is thrown when classes in this module are
```

Comment:
weird. I can take a look

Reply:
i think this is resolved in #269 fwiw; i don't remember what the solution was...maybe it was because some things got moved between modules or imports changed

Reply:
that makes sense! will be good to have it resolved